packages/client/src/index.ts

```diff
@@ -43,19 +53,68 @@ export const createClient = <schema extends Schema>(
   url: string,
   { schema }: { schema: schema },
 ): Client<schema> => {
-  const db = drizzle(
-    async (sql, params, method, typings) => {
-      const result = await fetch(`${url}/client`, {
-        method: "POST",
-        body: JSON.stringify({ sql, params, method, typings }),
-      });
-
-      return await result.json();
+  const noopDatabase = drizzle(
+    () => {
+      return Promise.resolve({ rows: [] });
     },
     { schema, casing: "snake_case" },
   );
 
-  return { db };
+  // @ts-ignore
+  const dialect: PgDialect = noopDatabase.dialect;
+
+  // @ts-ignore
+  const session: PgSession = noopDatabase.session;
+
+  return {
+    db: drizzle(
+      async (sql, params, method, typings) => {
+        const result = await fetch(`${url}/client/db`, {
```

Comment:
You might want to use the `URL` Web API here to handle cases where `url` has a trailing slash or already has query/search params.